### PR TITLE
feat: Improve logging styling.

### PR DESCRIPTION
**Pull request by xiCO2k**
This PR improves the Log output, with some more margins and paddings.

## Before and After:

image

PS: There is a need to have a [Termwind](https://github.com/nunomaduro/termwind) version on JS. 😎

### CODE REVIEW

**Comment by rennokki** [comment]
Amazing 🤩

File: src/log.ts [hunk 1]
```diff
@@ -80,4 +84,32 @@ export class Log {
...
+        const applyPadding = (message: string): string => {
+            const spaces = styles
+                .filter(style => /^px-/.test(style))
+                .map(style => ' '.repeat(parseInt(style.substr(3))))
+                .join('');
+
+            return spaces + message + spaces;
+        }
+
+        console.log(applyMargins(withColor(applyPadding(message))));
```
Comment (rennokki) [inline src/log.ts:113]: @xiCO2k It seems that this line is causing errors like in https://github.com/soketi/soketi/issues/451, any idea to workaround this? 🤔 I've tried `.dir()`, but it outputs weird stuff 

```bash
'  \x1B[36m🕵️‍♂️ Initiating metrics endpoints...\x1B[39m  '

'  \x1B[1m\x1B[30m\x1B[42m 🎉 Server is up and running! \x1B[49m\x1B[39m\x1B[22m  '
'  \x1B[1m\x1B[30m\x1B[42m 📡 The Websockets server is available at 127.0.0.1:6001 \x1B[49m\x1B[39m\x1B[22m  '      
'  \x1B[1m\x1B[30m\x1B[42m 🔗 The HTTP API server is available at http://127.0.0.1:6001 \x1B[49m\x1B[39m\x1B[22m  ' 
'  \x1B[1m\x1B[30m\x1B[42m 🎊 The /usage endpoint is available on port 9601. \x1B[49m\x1B[39m\x1B[22m  '
'  \x1B[1m\x1B[30m\x1B[42m 🌠 Prometheus /metrics endpoint is available on port 9601. \x1B[49m\x1B[39m\x1B[22m  '   
```

**Comment by xiCO2k** [comment]
Will check it out.

**Comment by xiCO2k** [comment]
Check https://github.com/soketi/soketi/pull/474

**Pull request merged**